### DISCUSSION TITLE
Try `fall` and `lag` commands

### DISCUSSION THREAD

**Discussion by winstxnhdw** [winstxnhdw/lc-hax#446] [Show and tell]
No idea if they work. I wrote this on my iPhone while taking a shit and I have no time to test this. In theory, it should work.

**Comment by Totoqoe** [comment]
I tested them and they don't seem to do anything, no one noticed any lag.  As for the fall command, it made the enemies invisible, but the others were still being chased by them as if it was still normal and the target player wasnt falling through the map.

**Reply by winstxnhdw** [reply]
Thanks for testing. I’ll look into it.

**Reply by winstxnhdw** [reply]
Can you try it again. Might've fixed it.

**Reply by Totoqoe** [reply]
Sorry about the wait, so far, fall did make a coilhead dissappear on others screens but it hasn't really caused anyone to fall through the map yet. Haven't gotten to test lag yet

**Reply by winstxnhdw** [reply]
> did make a coilhead dissappear on others screens but it hasn't really caused anyone to fall through the map yet

Okay, good to know it's finally showing up on their end. This means `lag` will likely work. Maybe `fall` requires a certain enemy instead of any. Have to test..

**Reply by Totoqoe** [reply]
Either people ignored it or `lag` is still not functioning properly, but its good to know that the enemies do dissappear for others.

**Reply by winstxnhdw** [reply]
Okay, for now I have tried teleporting the bracken directly to the player instead. This way their client should steal ownership, compute the broken path finding and start lagging. In my experience, their FPS should drop to 25 and they will get massive stutters.

**Reply by Totoqoe** [reply]
I tested this with a streamer and it didn't seem to cause any change performance wise.

**Reply by winstxnhdw** [reply]
Did the bracken appear at him while he was outside the factory?

**Reply by Totoqoe** [reply]
Nope, though we did play on a custom map so that might have broken the command?

**Reply by winstxnhdw** [reply]
Aha, if it appears on them and it still fails, I’ll just remove the command. Last week, I managed to get it to work on a friend but 1) I was host and 2) the Bracken was spawned with `/spawn` so I couldn’t use that information to reliably verify if it’s working.

**Comment by odzigon** [comment]
I improved lc-hax in order to crash the crew (by sending a lot of packets to the server), moreover the command works but it is necessary to work on it and also to make specific movements in the part so that the crew noclip in the map, moreover my cheat combined with lc-hax offers a perfect singularity.

**Comment by winstxnhdw** [comment]
I tested `fall` recently and it does work still but it doesn't always affect the target player. I am still not sure what are the exact conditions yet.
